### DISCUSSION TITLE
Custom model for SAHI

### DISCUSSION THREAD

**Discussion by Sarath-C-Koppolu** [ultralytics/ultralytics#7194] [General]
Hello Ultralytics team. 
Can you guide me to implement SAHI on custom model.

**Comment by RizwanMunawar** [comment]
@Sarath-C-Koppolu, are you talking about the custom model, specifically the 'model fine-tuned on a custom dataset using Ultralytics YOLOv8'?

Thanks

**Reply by Sarath-C-Koppolu** [reply]
Yes custom finetuned model 
But no Not only YOLOv8, 
but other models like RT-DETR.... ?

**Reply by RizwanMunawar** [reply]
@Sarath-C-Koppolu Custom model support has not been added yet! We have plans to incorporate this feature in the future. Currently, only Ultralytics YOLOv8 pretrained and fine-tuned models are supported.

Thanks
Ultralytics Team
